### PR TITLE
EV-69 stylelint config

### PR DESCRIPTION
**Pull request by LisovyiNazar**
### Jira

* [EV-69](https://chita.atlassian.net/jira/software/projects/EV/boards/1?selectedIssue=EV-69)

### Description
In this task must be implemented the integration of stylelint into the project, its configuration, and correction of all errors which it will find. Another task is to write documentation for stylelint

### Summary of change
Add stylelint to project
Correct all errors
Write the documentation.

### Checklist

- [x] Includes a link to JIRA ticket and all related PR (API dependencies etc.)
- [x] Includes a brief description of what has been changed
- [x] Test coverage for **new** code doesn't decrease the existing test coverage
- [x] Lint errors/warnings fixed
- [x] New code is documented if necessary
- [x] Dev Testing has been successfully finished
- [x] Reviewed by 2 developer on your team
- [x] Reviewed by 1 expert

### CODE REVIEW

File: src/components/AboutUs/InfoSection/InfoSection.scss [hunk 1]
```diff
@@ -9,37 +9,38 @@
...
 
             .info-images__item-img {
-                filter: brightness($infoImagesBrightness);
-                width: $fullWidth;
-                height: $fullHeight;
-                object-fit: cover;
                 position: absolute;
                 top: 0;
                 left: 0;
                 z-index: -1;
```
Comment (Danyil-man) [inline src/components/AboutUs/InfoSection/InfoSection.scss:39]: What is responsible for z-index: -1?

File: src/components/Admin/Admin.scss [hunk 2]
```diff
@@ -1,47 +1,49 @@
+@import "../../styles/colors.scss";
+
 body {
-  height: 100vh;
+    height: 100vh;
 }
 
 #root {
-  height: 100%;
+    height: 100%;
```
Comment (Danyil-man) [inline src/components/Admin/Admin.scss:8]: const

File: src/components/Admin/Admin.scss [hunk 3]
```diff
@@ -1,47 +1,49 @@
...
 }
 
 .admin-sidebar ul li .link {
-  padding: 10px;
-  font-size: 1.1em;
-  color: rgba(0, 0, 0, 0.75);
-  display: block;
-  width: 100%;
+    display: block;
+    width: 100%;
```
Comment (Danyil-man) [inline src/components/Admin/Admin.scss:33]: const

File: src/components/Admin/Admin.scss [hunk 4]
```diff
@@ -1,47 +1,49 @@
...
+    padding: 10px;
+    color: rgb(0 0 0 / 75%);
+    font-size: 1.1em;
```
Comment (Danyil-man) [inline src/components/Admin/Admin.scss:36]: const

File: src/components/Admin/Admin.scss [hunk 5]
```diff
@@ -1,47 +1,49 @@
...
 .admin-sidebar ul li .link {
-  padding: 10px;
-  font-size: 1.1em;
-  color: rgba(0, 0, 0, 0.75);
-  display: block;
-  width: 100%;
+    display: block;
+    width: 100%;
+    padding: 10px;
+    color: rgb(0 0 0 / 75%);
```
Comment (Danyil-man) [inline src/components/Admin/Admin.scss:35]: const

File: src/components/Admin/Admin.scss [hunk 6]
```diff
@@ -1,47 +1,49 @@
...
-  color: rgb(0, 0, 0);
-  background: #fff;
-  font-weight: 550;
-  border-bottom: 3px solid rgb(63, 81, 181);
-  transition: 0.1s;
+    color: rgb(0 0 0);
+    font-weight: 550;
+    background: $white;
+    border-bottom: 3px solid rgb(63 81 181);
+    transition: 0.1s;
```
Comment (Danyil-man) [inline src/components/Admin/Admin.scss:44]: use consts

File: src/components/Admin/Admin.scss [hunk 7]
```diff
@@ -1,47 +1,49 @@
...
 }
 
 #sub-nav div li a {
-  width: 100%;
+    width: 100%;
```
Comment (Danyil-man) [inline src/components/Admin/Admin.scss:48]: const

File: src/components/Chat/Msg.scss [hunk 8]
```diff
@@ -1,169 +1,199 @@
...
-	}
\ No newline at end of file
+.chat {
+    margin-top: auto;
+    margin-bottom: auto;
+}
+
+.card {
+    height: 95vh;
+    border-radius: 15px !important;
```
Comment (Danyil-man) [inline src/components/Chat/Msg.scss:10]: const

File: src/components/Chat/Msg.scss [hunk 9]
```diff
@@ -1,169 +1,199 @@
...
+
+.container {
+    align-content: center;
+}
+
+.search {
+    color: $white !important;
+    background-color: rgb(0 0 0 / 30%) !important;
+    border: 0 !important;
+    border-radius: 15px 0 0 15px !important;
```
Comment (Danyil-man) [inline src/components/Chat/Msg.scss:42]: use consts

File: src/components/Comment/Comment.scss [hunk 10]
```diff
@@ -19,25 +21,26 @@
 }
 
 .comment-container {
-    width: 100%;
-    margin-bottom: 5px;
     display: flex;
     flex-direction: row;
     justify-content: space-between;
+    width: 100%;
```
Comment (Danyil-man) [inline src/components/Comment/Comment.scss:27]: use mixin or consts

File: src/components/Comment/Comment.scss [hunk 11]
```diff
@@ -19,25 +21,26 @@
...
 .ItemComment {
-    border-radius: 15px;
     display: flex;
+    justify-content: space-between;
     padding: 15px;
     padding-bottom: 0;
-    color: black;
-    justify-content: space-between;
+    color: $black;
+    border-radius: 15px;
```
Comment (Danyil-man) [inline src/components/Comment/Comment.scss:37]: const

File: src/components/ContactAdmin/ContactAdminFilter.scss [hunk 12]
```diff
@@ -1,30 +1,28 @@
 .sidebar-filter {
     position: fixed;
+    top: inherit;
     right: 0;
-    top:inherit;
+    width: 15em;
+    height: 100%;
```
Comment (Danyil-man) [inline src/components/ContactAdmin/ContactAdminFilter.scss:6]: const

File: src/components/ContactAdmin/ContactAdminFilter.scss [hunk 13]
```diff
@@ -1,30 +1,28 @@
...
-    width: 15em;   
-    height: 100%;
     overflow: auto;
-  }
-  
-.events-container{
+}
+
+.events-container {
     width: 100%;
```
Comment (Danyil-man) [inline src/components/ContactAdmin/ContactAdminFilter.scss:13]: const

File: src/components/Event/EventFilter/EventFilter.scss [hunk 14]
```diff
@@ -1,30 +1,28 @@
...
-      height: auto !important;
-      position: relative !important;
+        position: relative !important;
+        width: 100% !important;
+        height: auto !important;
     }
-    
-    .events-container{
+
+    .events-container {
```
Comment (Danyil-man) [inline src/components/Event/EventFilter/EventFilter.scss:25]: use consts

File: src/components/Event/EventForm/EventForm.scss [hunk 15]
```diff
@@ -63,16 +66,16 @@
...
-    width: 100%;
-    height: 100%;
-    margin: 0;
     position: relative;
-    text-align: center;
     display: flex;
     align-items: center;
     justify-content: center;
+    width: 100%;
+    height: 100%;
```
Comment (Danyil-man) [inline src/components/Event/EventForm/EventForm.scss:74]: consts

File: src/components/Event/EventForm/EventForm.scss [hunk 16]
```diff
@@ -89,11 +92,11 @@
...
 
 .pic-overlay .fa-window-close-o {
-    color: red;
+    color: $deleteColor;
 }
 
 .pic-overlay .fa-pencil-square-o {
-    color: blue;
+    color: $blue;
     font-size: 40px;
```
Comment (Danyil-man) [inline src/components/Event/EventForm/EventForm.scss:100]: const

File: src/components/Event/EventItemView/EventItemView.scss [hunk 17]
```diff
@@ -1,74 +1,82 @@
...
-  font-size: 28px;
-  padding-left: 20px;
-  padding-right: 20px;
+    position: absolute;
+    bottom: 20px;
+    left: 20px;
+    padding-right: 20px;
+    padding-left: 20px;
+    color: $white;
+    font-size: 28px;
```
Comment (Danyil-man) [inline src/components/Event/EventItemView/EventItemView.scss:10]: const

File: src/components/Event/EventItemView/EventItemView.scss [hunk 18]
```diff
@@ -1,74 +1,82 @@
...
 }
 
 .join-leave {
     font-size: 20px;
 }
 
 .btn-edit {
-  font-size: 24px;
-  background: #da2121ab;
+    font-size: 24px;
```
Comment (Danyil-man) [inline src/components/Event/EventItemView/EventItemView.scss:32]: use consts

File: src/components/Event/EventItemView/EventItemView.scss [hunk 19]
```diff
@@ -1,74 +1,82 @@
...
 
 .text-box-big {
-    overflow-wrap: anywhere;    
+    overflow-wrap: anywhere;
     scroll-behavior: auto;
 }
+
 .cancel-text {
-  font-size: 24px;
+    font-size: 24px;
```
Comment (Danyil-man) [inline src/components/Event/EventItemView/EventItemView.scss:56]: const

File: src/components/Event/EventItemView/EventItemView.scss [hunk 20]
```diff
@@ -1,74 +1,82 @@
...
-    display: none;
     position: absolute;
-    background-color: #f9f9f9;
-    min-width: 160px;
-    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
     z-index: 1;
+    display: none;
+    min-width: 160px;
+    background-color: $white;
+    box-shadow: 0 8px 16px 0 rgb(0 0 0 / 20%);
```
Comment (Danyil-man) [inline src/components/Event/EventItemView/EventItemView.scss:76]: const

File: src/components/Event/EventsForAdmin/EventsForAdmin.scss [hunk 21]
```diff
@@ -1,10 +1,11 @@
 .admin-events-container {
-  width: 100%;
-  padding-right: 15em;
+    width: 100%;
```
Comment (Danyil-man) [inline src/components/Event/EventsForAdmin/EventsForAdmin.scss:2]: const

File: src/components/Event/ImageResizer/ImageResizer.scss [hunk 22]
```diff
@@ -12,22 +12,22 @@
...
     bottom: 0;
     left: 50%;
-    width: 50%;
-    transform: translateX(-50%);
-    height: 80px;
     display: flex;
     align-items: center;
+    width: 50%;
+    height: 80px;
+    transform: translateX(-50%);
```
Comment (Danyil-man) [inline src/components/Event/ImageResizer/ImageResizer.scss:28]: consts

File: src/components/Event/Map/Map.scss [hunk 23]
```diff
@@ -1,20 +1,22 @@
...
-    text-align: left;
-    font-family: "Roboto", "Helvetica", "Arial", sans-serif;
+    margin-top: 8px;
+    color: $redOrange;
     font-weight: 400;
+    font-size: 0.75rem;
+    font-family: Roboto, Helvetica, Arial, sans-serif;
     line-height: 1em;
-    letter-spacing: 0.03333em;
+    letter-spacing: 0.0333em;
```
Comment (Danyil-man) [inline src/components/Event/Map/Map.scss:16]: consts

File: src/components/Event/Share/Share.scss [hunk 24]
```diff
@@ -1,33 +1,35 @@
...
-    border-radius: 3px;
-    font-weight: 600;
-    padding: 8px 20px;
-    display: inline-block;
     position: static;
+    display: inline-block;
+    padding: 8px 20px;
+    font-weight: 600;
+    background: $lightBlue;
+    border-radius: 3px;
```
Comment (Danyil-man) [inline src/components/Event/Share/Share.scss:15]: consts

File: src/components/Event/Share/Share.scss [hunk 25]
```diff
@@ -1,33 +1,35 @@
...
-    #fb-share-button span {
-        vertical-align: middle;
-        color: white;
-        font-size: 14px;
-        padding: 0 3px
-    }
+#fb-share-button span {
+    padding: 0 3px;
+    color: $white;
+    font-size: 14px;
```
Comment (Danyil-man) [inline src/components/Event/Share/Share.scss:33]: const

File: src/components/Event/Slider.scss [hunk 26]
```diff
@@ -1,40 +1,42 @@
...
+    background: $middleBlue;
     border-radius: 50%;
-    background: #4d7bdd;
     cursor: pointer;
-  }
-  .slidecontainer {
+}
+
+.slidecontainer {
     width: 100%;
```
Comment (Danyil-man) [inline src/components/Event/Slider.scss:41]: use conts

File: src/components/Header/DropdownMenu/DropdownMenu.scss [hunk 27]
```diff
@@ -3,56 +3,57 @@
...
 }
 
 .dropdown-toggle {
-  align-items: center;
-  display: flex;
-  flex-direction: row;
-  color: $black;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
```
Comment (Danyil-man) [inline src/components/Header/DropdownMenu/DropdownMenu.scss:21]: As i know< we have mixin for it

File: src/components/Header/MainHeader.scss [hunk 28]
```diff
@@ -2,36 +2,36 @@
...
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  padding: 2px 5px;
+    position: fixed;
+    top: 0;
+    z-index: 50;
+    display: flex;
+    align-items: center;
+    justify-content: center;
```
Comment (Danyil-man) [inline src/components/Header/MainHeader.scss:10]: mixin

File: src/components/Inventory/Inventory.scss [hunk 29]
```diff
@@ -1,44 +1,48 @@
...
-  border-radius: 19px;
-  text-align: center;
-  padding-left: 0;
-  padding-right: 0;
-  font-size: 16px;
+    max-width: 38px;
+    height: 38px;
+    padding-right: 0;
+    padding-left: 0;
+    font-size: 16px;
```
Comment (Danyil-man) [inline src/components/Inventory/Inventory.scss:9]: const

File: src/components/Landing/EventsViewMode/EventsViewMode.scss [hunk 30]
```diff
@@ -1,33 +1,34 @@
...
+@import "../../../styles/variables.scss";
 
 .menu-view-mode {
-  margin: 20px 0px;
-  display: flex;
-  justify-content: end;
-  align-items: center;
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
```
Comment (Danyil-man) [inline src/components/Landing/EventsViewMode/EventsViewMode.scss:6]: mixin

File: src/components/Profile/UserProfile.scss [hunk 31]
```diff
@@ -1,95 +1,99 @@
-﻿
+@import "../../styles/colors.scss";
+@import "../../styles//fonts.scss";
 
 .funkyradio label {
     width: 100%;
```
Comment (Danyil-man) [inline src/components/Profile/UserProfile.scss:5]: const

File: src/components/Register/Register.scss [hunk 32]
```diff
@@ -1,3 +1,3 @@
 .registerFormInputs {
-  width: 100%;
+    width: 100%;
```
Comment (Danyil-man) [inline src/components/Register/Register.scss:2]: const

File: src/components/RouteGuard/RouteGuardErrors.scss [hunk 33]
```diff
@@ -1,38 +1,41 @@
...
-    box-shadow: 0.125rem 0.125rem 0.25rem rgba(0, 0, 0, 0.25);
     display: flex;
-    font-family: Helvetica, sans-serif;
     justify-content: space-between;
-    padding: 1rem;
-    margin: 1rem auto;
     width: 25rem;
+    margin: 1rem auto;
+    padding: 1rem;
+    font-family: Helvetica, sans-serif;
```
Comment (Danyil-man) [inline src/components/RouteGuard/RouteGuardErrors.scss:15]: const

File: src/components/SearchInput/SearchInput.scss [hunk 34]
```diff
@@ -2,43 +2,43 @@
 @import "../../styles/colors.scss";
 @import "../../styles/fonts.scss";
 
-
 .MainSearchWrapper {
-    width: $fullWidth;
     display: flex;
     align-items: center;
     justify-content: center;
```
Comment (Danyil-man) [inline src/components/SearchInput/SearchInput.scss:8]: use consts

File: src/components/Stepper/Stepper.scss [hunk 35]
```diff
@@ -1,60 +1,60 @@
...
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  position: relative;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
```
Comment (Danyil-man) [inline src/components/Stepper/Stepper.scss:11]: mixin

File: src/components/Tile/Tile.scss [hunk 36]
```diff
@@ -1,37 +1,39 @@
...
-  cursor: pointer;
+@import "../.././styles/colors.scss";
+
+.tile {
+    display: flex;
+    flex-direction: row-reverse;
+    align-items: flex-start;
+    min-height: 16vh;
+    background-color: $gray;
+    border-radius: 12px;
```
Comment (Danyil-man) [inline src/components/Tile/Tile.scss:9]: mixin, const for border

File: src/containers/TileRow/CustomCheckbox.scss [hunk 37]
```diff
@@ -1,57 +1,60 @@
...
+    margin: 0 2.5vw 0 0.5vw;
 }
 
 .checkbox > label {
-  font-size: 1.5rem;
-  font-weight: bold;
-  line-height: 1.3;
-  vertical-align: sub;
+    font-weight: $bold;
+    font-size: 1.5rem;
```
Comment (Danyil-man) [inline src/containers/TileRow/CustomCheckbox.scss:10]: const

File: src/containers/TileRow/Line.scss [hunk 38]
```diff
@@ -1,11 +1,13 @@
+@import "../../styles/colors.scss";
+
 .line {
     display: block;
     width: 100%;
```
Comment (Danyil-man) [inline src/containers/TileRow/Line.scss:5]: const

**Review by Danyil-man — COMMENTED** [review]
Fix conversations

File: src/components/AboutUs/InfoSection/InfoSection.scss [hunk 39]
```diff
@@ -9,37 +9,38 @@
     padding: 50px;
```
Comment (PopowychSasha) [inline src/components/AboutUs/InfoSection/InfoSection.scss:9]: use constants

File: src/components/Admin/Admin.scss [hunk 40]
```diff
@@ -1,47 +1,49 @@
+@import "../../styles/colors.scss";
+
 body {
-  height: 100vh;
+    height: 100vh;
```
Comment (PopowychSasha) [inline src/components/Admin/Admin.scss:4]: use existing const

File: src/components/Admin/Admin.scss (hunk 2 above)
Comment (PopowychSasha) [inline src/components/Admin/Admin.scss:8]: use existing const

File: src/components/Chat/Msg.scss [hunk 41]
```diff
@@ -1,169 +1,199 @@
...
+.type_msg:focus {
+    outline: 0 !important;
+    box-shadow: none !important;
+}
+
+.attach_btn {
+    color: $white !important;
+    background-color: rgb(0 0 0 / 30%) !important;
+    border: 0 !important;
+    border-radius: 15px 0 0 15px !important;
```
Comment (PopowychSasha) [inline src/components/Chat/Msg.scss:67]: Can you set style without !important?

File: src/components/ContactAdmin/ContactAdminFilter.scss (hunk 13 above)
Comment (PopowychSasha) [inline src/components/ContactAdmin/ContactAdminFilter.scss:13]: const

File: src/components/Event/EventForm/EventForm.scss (hunk 15 above)
Comment (PopowychSasha) [inline src/components/Event/EventForm/EventForm.scss:74]: const

**Pull request closed without merging**